js/utils.js

```diff
@@ -17,6 +17,50 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
   };
   var computedFieldClashes = [];
 
+  function setLineClamps (selector) {
+    return setClampHeight(selector)
+      .then(function (elements) {
+        Superclamp.register(elements);
+        removeClampHeight(elements);
+      })
+      .catch(function (selector) {
+        console.error('Fail to clamp line in the selector ' + selector);
+      });
+  }
+
+  function setClampHeight (selector) {
+    selector = selector || '[data-line-clamp]';
+
+    return new Promise(function (resolve, reject) {
+      var $elements = $(selector);
+      
+      if (!$elements.length) {
+        reject(selector);
+      }
+
+      $elements.each(function () {
+        var element = this;
+        var lineHeightValue = window.getComputedStyle(element)
+          .getPropertyValue('line-height').match(/[a-zA-Z]+|[0-9]+(?:\.[0-9]+|)/g);
+        var lineHeightMeasure  = lineHeightValue[1];
+        var elementHeight = parseFloat(lineHeightValue[0]);
+        var clampLines = element.dataset.lineClamp;
+        var elementHeight = elementHeight * clampLines;
+
+        $(element).css('height', elementHeight + lineHeightMeasure );
+      });
+
+      resolve($elements);
+    });
+  }
+
+  // To avoid large distance between block we remove height at the element
+  function removeClampHeight ($elements) {
+    $elements.each(function() {
+      $(this).css('height', 'auto');
```

Comment:
This is the better way to remove the `height` property instead of setting it to `auto`. That's because if there is CSS setting the height to, e.g. `3em`, then removing the clamp height should reset it to `3em` by removing the inline CSS, instead of changing `3em` to `auto`.

```suggestion
      $(this).css('height', '');
```